spring-cloud-gcp-data-spanner/README.adoc

```diff
@@ -0,0 +1,72 @@
+:ext-relative: adoc
+
+= image:spanner-logo.png[50,50] Spring Data Spanner
+
+Provides a http://projects.spring.io/spring-data/[Spring Data Module] for https://cloud.google.com/spanner/[Google Cloud Spanner].
+
+Maven coordinates for this module only, using Spring Cloud GCP BOM:
+
+[source,xml]
+----
+<dependency>
+    <groupId>org.springframework.cloud</groupId>
+    <artifactId>spring-cloud-gcp-data-spanner</artifactId>
+</dependency>
+----
+
+Gradle coordinates:
+
+[source]
+----
+dependencies {
+    compile group: 'org.springframework.cloud', name: 'spring-cloud-gcp-data-spanner'
+}
+----
+
+With our link:../spring-cloud-gcp-starters/spring-cloud-gcp-starter-data-spanner[Spring Boot Starter for Spring Data Spanner], you can leverage our recommended auto-configuration setup. (see https://docs.spring.io/spring-boot/docs/current/reference/htmlsingle/#using-boot-starter[Spring Boot Starters] for more details on what a starter is).
+To use the starter, see the coordinates see below.
+
+[source,xml]
+----
+<dependency>
+    <groupId>org.springframework.cloud</groupId>
+    <artifactId>spring-cloud-gcp-starter-data-spanner</artifactId>
+</dependency>
+----
+
+Gradle coordinates:
+
+[source]
+----
+dependencies {
+    compile group: 'org.springframework.cloud', name: 'spring-cloud-gcp-starter-data-spanner'
+}
+----
+
+==== What's new?
+
+Spring Spanner Data is considered experimental stability.
+
+Features to be released with 1.0.0.M3:
+
+- Mapping POJO domain objects to Spanner tables using annotations
+- SpannerTemplate API:
+   ** Stale read
+   ** Read with secondary indices
+   ** Read with limits and flow control
+- Spring Data Repositories: CrudRepository, PagingAndSortingRepository
+- Declarative query methods
+   ** interface method resolution by naming convention
+   ** read-only SQL via properties files
+- basic Spanner types are support in fields
```

Comment:
"are supported" 

But do you mean by this that we only support the basic spanner types (cuz we do have custom conversions and common java types handled now)

Reply:
thanks, I'll update it - this was before the feature was decided on! :)